docs(ReadMe): Updated readMe & indexed fields on events. (#267)
WavesHQ/bridge: README.md

To change the state of any smart contract, users need to approve the smart contract of the respective token via the `approve()` function first. Once approved, user will be able to bridge the token over to DefiChain.

### Bridge ERC20 tokens - to transfer ERC20 tokens from an EOA to the Bridge

Once approved, user will call the `bridgeToDeFiChain()` function with following arguments: `_defiAddress`- address on Defi Chain that receiving funds, `_tokenAddress` - ERC20 token's address and `_amount` amount to bridge over to Defi chain.

> ### Bridge Ether - to transfer ETHER from an EOA to the Bridge

Reviewer: This section can be removed too after #276 🤣

### BridgeProxy

BridgeProxy Contract addrress: [0x93fE70235854e7c97A5db5ddfC6eAAb078e99d3C](https://goerli.etherscan.io/address/0x93fE70235854e7c97A5db5ddfC6eAAb078e99d3C)

> ## Fund Bridge with ETHER and ERC20 tokens

Reviewer: There's no more ETH, only wETH 😛

Author: I removed this on PR #289

> ### Add funds

Author: @cuongquangnam , can you please quickly double check this?

> Anyone can send fund to the bridge contract, ideally in this case, this should be done by liquidity providers. If tokens send by other addresses straight to the contract, those tokens will be unaccounted for. In this case, Admin will have to manually process the refund.

Reviewer: ```suggestion
Anyone can send funds to the bridge contract. Ideally, this should be done by liquidity providers. If there are tokens sent by other addresses to the contract, those tokens will be unaccounted for.
```

Removing the part about processing refunds. Such cases will be the exception - not the norm.

> Admins can send the ETHER and ERC20 tokens via the `transfer(address _to, uint256 _amount)` function.

Reviewer 2: for the case of ether, it will be a simple transfer in metamask

Author: Yes, however the underlying function will be `transfer` regardless of ETHER and ERC20 via metamask

> Admins can send the ETHER and ERC20 tokens via the `transfer(address _to, uint256 _amount)` function or utilizing wallet such as Metamask.

Reviewer: ```suggestion
Admins can send the ERC20 tokens via the `transfer(address _to, uint256 _amount)` function or utilizing wallets such as Metamask.
```

> Funding the bridge contract would ideally be done on lunch (Not decided on specific numbers)

Reviewer 2: on launch right, at first sight, I thought we gonna do funding while eating

Author: oh that's one of a kind typo hahaha

> Funding the bridge contract would ideally be done upon launch (Not decided on specific numbers)

Reviewer: ```suggestion
```

No need to mention this!

> ### Withdraw funds
> 
> ETHER and ERC20 can be withdrawn by the Admin address only via `withdrawEth(uint256 amount)` and `withdraw(address _tokenAddress, uint256 amount)` functions respectively.

Reviewer: ```suggestion
### Withdrawing funds

ERC20 tokens can be withdrawn by the Admin address only via the `withdraw(address _tokenAddress, uint256 amount)` function.
```

Reviewer: This will need to be updated when #277 implementation is added.

## Admin and Operational addresses - Gnosis safe

> Admin and Operational addresses will be Gnosis safes, ideally will be with at least 3 owners. In case of 3 addresses, 2 approval ( this can be set when making a safe or later ) will be needed to execute txs.

Reviewer: ```suggestion
Admin and Operational addresses will be Gnosis safes, ideally will be with at least 3 owners with a 2/3 quorum.
```

> More admins can be added later, for more info [Gnosis safe: adding owners](https://help.gnosis-safe.io/en/articles/3950657-add-owners).

Reviewer: ```suggestion
More admins can be added later, for more info, refer to [Gnosis safe: adding owners](https://help.gnosis-safe.io/en/articles/3950657-add-owners).
```
